src/WebJobs.Script/Binding/WebJobsCoreScriptBindingProvider.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.IO;
+using Microsoft.Azure.WebJobs.Script.Extensibility;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    /// <summary>
+    /// Enables all Core SDK Triggers/Binders
+    /// </summary>
+    [CLSCompliant(false)]
+    public class WebJobsCoreScriptBindingProvider : ScriptBindingProvider
```

Comment:
This one CANT be moved into SDK core, since that would create a circular assembly dependency (since WebJobs.Script.Extensibility references Microsoft.Azure.WebJobs.Host).